starlette/types.py

```diff
@@ -9,3 +9,9 @@
 
 ASGIInstance = typing.Callable[[Receive, Send], typing.Awaitable[None]]
 ASGIApp = typing.Callable[[Scope], ASGIInstance]
+
+StrDict = typing.Mapping[str, str]
+BytesPairs = typing.List[typing.Tuple[bytes, bytes]]
+
+Methods = typing.Sequence[str]
+Subprotocols = typing.Sequence[str]
```

Comment:
Let's avoid these extra indirections, and just use the full type annotations directly in these 4 cases.